src/pocketmine/entity/Entity.php

```diff
@@ -978,6 +979,13 @@ protected function initEntity() : void{
 			}else{
 				$this->setNameTagVisible($this->namedtag->getByte("CustomNameVisible", 1) !== 0);
 			}
+			
+			if($this->namedtag->hasTag("CustomNameAlwaysVisible", StringTag::class)){
+                $this->setNameTagAlwaysVisible($this->namedtag->getString("CustomNameAlwaysVisible") !== "");
+                $this->namedtag->removeTag("CustomNameAlwaysVisible");
+            }else{
+			    $this->setNameTagAlwaysVisible($this->namedtag->getByte("CustomNameAlwaysVisible", 1) !== 0);
```

Comment:
Always visible is not the default. Again, obvious copy paste job.

Reply:
should i add the bytetag checking?